### PR TITLE
Tweak Join Lines command

### PR DESCRIPTION
**Pull request by ntrel**
Add menu item.
Don't require line breaking or line marker to be set.

### CODE REVIEW

**Comment by elextr** [comment]
LGBI

**Comment by ntrel** [comment]
Added docs.

File: doc/geany.txt [hunk 1]
```diff
@@ -3624,6 +3624,12 @@ Reflow lines/block                                        Reformat selected line
                                                           breaking lines at the long line marker or the
                                                           line breaking column if line breaking is
                                                           enabled for the current document.
+
+Join Lines                                                If no selection, remove line endings and indentation
+                                                          after the current line until the end of the current
+                                                          (indented) text block.
```
Comment (b4n) [inline doc/geany.txt:3630]: I'm not sure if this is entirely clear that it means "the block of lines at the current indentation level, separated by blank lines (indented or not)".

**Review by b4n — APPROVED** [review]
Apart perhaps finding better wording for the documentation, works well and does what it promises.

File: doc/geany.txt (hunk 1 above)
Comment (ntrel) [reply doc/geany.txt]: Thanks, updated.

**Pull request merged**